presto-main/src/main/resources/webapp/src/components/PageTitle.jsx

```diff
@@ -92,7 +92,7 @@ export class PageTitle extends React.Component {
                                 <tbody>
                                 <tr>
                                     <td>
-                                        <a href="/"><img src="assets/logo.png"/></a>
+                                        <a href="/ui/"><img src="assets/logo.png"/></a>
```

Comment:
The previous link should work as well

Reply:
@raghavsethi is the purpose of this change to ease relocation/embedding of the page?

Reply:
It should, technically, but it does not work in all cases (some proxies mangle things)

Reply:
@ggreg It's not intended for that, since this component will likely not be embedded.